public/js/rules.js

```diff
@@ -0,0 +1,79 @@
+var Rule = React.createClass({
+    propTypes: {
+        name: React.PropTypes.string.isRequired,
+        contents: React.PropTypes.object.isRequired
+    },
+    render: function() {
+        var contentNodes = this.props.contents.map(function(c) {
+            return (
+                // Get raw data so any html formatting in the model isn't lost.
+                // eslint-disable-next-line react/no-danger
+                <li dangerouslySetInnerHTML={{__html:c.toString()}} />
```

Comment:
FYI: only disabling react/no-danger for the one line since dangerouslySetInnerHTML isn't something we want being used very often. Of course here the data comes from our side and hence we can ensure the sanity of any html inside of it.